src/main/java/com/example/application/ui/horizontal/library/ArchiveView.java

```diff
@@ -18,36 +19,30 @@
 /**
  *  Archive View shows a short description and a non-functional button
  *
- *  @author Vanessa Skowronsky
- *  @version 2.0
+ *  @author Vanessa Skowronsky and Laura Neuendorf
```

Comment:
Do not change author, only who @ last updated

Reply:
Hab es wie Jessica immer so gemacht. Da war die Kommunikation wohl irgendwie falsch. Wird in den anderen Klassen auch so sein.. Soll ich das einfach mit dem zweiten Teil ändern?

Reply:
Du kannst es auch in einem nächsten Pull Request ändern, Jessica weiß auch jetzt schon Bescheid, wie das gemacht werden soll. Wir ignorieren es jetzt :D